app/views/giraffe/thankyou.scala.html

```diff
@@ -10,9 +10,12 @@ <h1 class="support-thanks__title">@pageInfo.title</h1>
             <p class="support-thanks__message">@pageInfo.description</p>
 
             <h1 class="support-thanks__title">Spread the word</h1>
-            <p class="support-thanks__message">
+            <p class="support-thanks__message" data-shown="gbp eur usd nzd cad">
                 We report for everyone. Do tell your friends and family how they can support the Guardian too.
             </p>
+            <p class="support-thanks__message" data-shown="aud">
+                Your contribution is much appreciated, and will help us to maintain our fearless, independent, journalism in Australia.
+            </p>
```

Comment:
As you mentioned yesterday, it's a bit weird that this is done client-side! Oh well...